dashboard/db/migrate/20160802170542_add_readonly_unlocked_at_to_user_levels.rb

```diff
@@ -0,0 +1,6 @@
+class AddReadonlyUnlockedAtToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :view_answers, :boolean
+    add_column :user_levels, :unlocked_at, :datetime
+  end
+end
```

Comment:
Yes, I'd recommend breaking it out into a separate PR so that you can run it in isolation on production after hours, when we disable writes to UserLevel using the queueing mechanism, and draining it after the migration completes.  We went through something similar with https://github.com/code-dot-org/code-dot-org/pull/6973/files